docker.py

```diff
@@ -84,7 +84,7 @@ def get_numpy_choices():
 
 RUN yum -y update && \\
     yum -y install epel-release && \\
-    yum -y install gcc gcc-c++ git kmod hdf5-devel perl make autoconf xz && \\
+    yum -y install gcc gcc-c++ cmake git kmod hdf5-devel perl make autoconf xz && \\
```

Comment:
iDeep is provided as wheel, so `cmake` is not required.

Reply:
Fixed.